src/lib.rs

```diff
@@ -447,12 +461,13 @@ impl Toast {
 
     // HACK: f is static so that we know the function is valid to call.
     //       this would be nice to remove at some point
-    pub fn on_activated<F: FnMut(Option<String>) -> Result<()> + Send + 'static>(
-        mut self,
-        mut f: F,
-    ) -> Self {
+    pub fn on_activated<F>(mut self, mut f: F) -> Self
```

Comment:
any reason to change the closure signature?

Reply:
while i was testing it, the compiler was giving error about windows::core::Result not being found, so i removed it and it was working fine. But i reversed this change in recent commit.